readthedocs/organizations/templates/organizations/organization_choose.html

```diff
@@ -0,0 +1,51 @@
+{% extends "base.html" %}
+
+{% load i18n %}
+{% load gravatar %}
+{% load organizations %}
+
+{% block title %}{% trans "Before continuing, choose an organization..." %}{% endblock %}
```

Comment:
The additional text isn't 
I'd say this sounds better without the passive clause in front, here and in the heading.

```suggestion
{% block title %}{% trans "Choose an organization" %}{% endblock %}
```